packages/react-checkbox/src/Checkbox.Circular.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { Checkbox } from './index';
+
+export const Circular = () => <Checkbox circular label="Circular" />;
```

Comment:
Do you think we'll have any other shapes in the future?  Square, rounded? If so, then we should use a discriminated union rather than a boolean.

Reply:
While I doubt we'll see additional shapes in the future I do like the idea of updating the API to accommodate the possibility. It doesn't really make the component API or implementation more complicated and makes potential future changes simpler.

Reply:
This prop is what was approved in the spec. We could make a PR to update the spec and get feedback on that; but that's out of scope of this.

Reply:
Logged #20993 to follow up

Reply:
we should make this update before this component is elevated to RC since removing the `circular` prop would be a breaking change.

Generally shape props are all discriminated string unions which was what @GeoffCoxMSFT  updated for all of our active beta components. Let's be consistent